WooCommerce/Classes/ViewRelated/Products/ProductSelector/ProductRowViewModel.swift

```diff
@@ -495,7 +508,11 @@ final class ProductRowViewModel: ObservableObject, Identifiable {
     ///
     func decrementQuantity() {
         guard quantity > minimumQuantity else {
-            return removeProductIntent()
+            if let removeProductIntent {
+                return removeProductIntent()
+            } else {
+                return
+            }
```

Comment:
super nit: could this be simplified as:

```suggestion
            return removeProductIntent?()
```